datalake/networking_stack.py

```diff
@@ -4,6 +4,9 @@
 from aws_cdk import aws_ec2, core
 from aws_cdk.core import Tags
 
+APPLICATION_NAME_TAG_NAME = "ApplicationName"
```

Comment:
I would separate changes in this file to other PR as well

Reply:
@l0b0 , because of time pressure, leave this change as is but I still think this is completely unrelated change to the scope of this PR and should be separated.